src/family/family.resolver.ts

```diff
@@ -16,4 +17,9 @@ export class FamilyResolver {
   findOne(@Args('id', { type: () => String }) id: string) {
     return this.familyService.findById(id);
   }
+
+  @Mutation(() => Family)
+  createFamily(@Args('data') createFamilyInput: CreateFamilyInput) {
```

Comment:
I think `input` is a better name than `data` (what is your opinion?)
I have used `input` 
for consistency it's better to use one of them

Reply:
I agree with input too.